client/http_curl.cpp

```diff
@@ -89,6 +89,7 @@ size_t libcurl_write(void *ptr, size_t size, size_t nmemb, HTTP_OP* phop) {
         );
     }
     phop->bytes_xferred += (double)(stWrite);
+    phop->update_http_ip_version();
```

Comment:
This function will write to the log file om every write call. This is not needed.
This should be written only once when connections is established